marimo/_messaging/ops.py

```diff
@@ -153,9 +153,11 @@ def __post_init__(self) -> None:
 
     @staticmethod
     def maybe_truncate_output(
-        mimetype: KnownMimeType, data: str
+        mimetype: KnownMimeType,
+        data: str,
+        output_max_bytes: int = OUTPUT_MAX_BYTES,
```

Comment:
ideally we'd remove the default value.  I will check the other call sites to see if I can retrieve a config object there.

Or is it always possible to call `get_context()` inside of this function, so `output_max_bytes` doesn't even have to be an argument?